kinder/pkg/actions/cluster-info.go

```diff
@@ -69,12 +69,18 @@ func runInfo(kctx *kcluster.KContext, kn *kcluster.KNode, flags kcluster.ActionF
 		return err
 	}
 
-	ip, err := kn.IP()
-	if err != nil {
-		return errors.Wrap(err, "Error getting node ip")
-	}
-
 	if kctx.ExternalEtcd() == nil {
+		// Gets the IP address where local etcd is listening for clients
+		// NB. before v1.13 local etcd is listening on local host; after v1.13
+		// local etcd is listening on the advertise address
+		ip := "127.0.0.1"
```

Comment:
possibly add a TODO that we should remove the hack once the problem version goes out of support?